common/src/main/java/com/netflix/conductor/common/metadata/tasks/TaskDef.java

```diff
@@ -113,6 +113,9 @@
 	@Email(message = "ownerEmail should be valid email address")
 	private String ownerEmail;
 
+	@ProtoField(id = 19)
+	private Integer pollTimeoutSeconds;
```

Comment:
This field shouldn't be allowed to be negative. Could you please add a `@Min` like for `responseTimeoutSeconds` ?

Reply:
Done